docs/products/opensearch/dashboards/getting-started.rst

```diff
@@ -0,0 +1,119 @@
+Getting started
+===============
+
+To start using Aiven for OpenSearch Dashboards, you should :doc:`create Aiven for OpenSearch service first</docs/products/opensearch/getting-started>` and OpenSearch Dashboards service will be created alongside it. Once the Aiven for OpenSearch service is running you can find connection information to your OpenSearch Dashboards in the service overview page.
+
+.. note::
+    If you are using Chrome, you need to enter **user name** and **password** manually. :doc:`Read more <howto/access-from-chrome>` for details.
+
+Load sample data
+*****************
+
+OpenSearch Dashboards come with three demonstration datasets included. To add sample data follow these steps:
+
+#. On the OpenSearch Dashboards landing page click on **Add sample data**.
+#. Choose one of the available datasets and click **Add data**.
+#. Click on **View data** to open the dashboard.
+
+Tools and pages
+***************
+
+OpenSearch Dashboards includes various tools to work with data and run queries in different ways. Below you can find ideas what you can do at different pages.
+
+Discover
+---------
+
+**Discover** page provides an interface to work with available data fields and run search queries by using either `the OpenSearch Dashboards Query Language (DQL) <https://opensearch.org/docs/latest/dashboards/dql/>`_  or Lucene.
+
+Additionally to search queries, you can filter the data by using either a visual interface or `OpenSearch Query DSL <https://opensearch.org/docs/latest/opensearch/query-dsl/index/>`_
+
+.. note::
+    If the index you're looking at contains a date field, pay attention to currently selected date range when running a query.
+
+
+Visualize
+----------
+**Visualize** page contains tools to manage your visualisations. In order to create a new visualization:
+
+#. Select visualization type you want to use.
+#. Choose the source of data.
+#. Follow the interface to set up metrics and buckets.
+
+Dashboard
+---------
+
+A set of visualization can be put together on a single dashboard. Search queries and filters applied to the dashboard will refine results for every included visualisation.
```

Comment:
"put" -> "organised" ?

Reply:
I feel that "put together" is a stable phrasal verb, it means "make something by assembling different parts" and sounds simple when reading.